docs/docs/schema/pipeline.json

```diff
@@ -288,7 +288,7 @@
                 },
                 "type": {
                     "default": "kafka-connector",
-                    "description": "Base class for all Kafka connectors\nShould only be used to set defaults",
+                    "description": "Used for schema generation, same as :param:`type`",
```

Comment:
isn't that the description of the `schema_type` attribute?

Reply:
Oof, yeah, idk what I was thinking...

[noted](https://github.com/bakdata/kpops/pull/267#discussion_r1252237204)